src/client/dfuse/dfuse_main.c

```diff
@@ -217,21 +217,64 @@ dfuse_launch_fuse(struct dfuse_projection_info *fs_handle,
 	return false;
 }
 
+static void
+show_version(char *name)
+{
+	fprintf(stdout, "%s version %s, libdaos %d.%d.%d\n",
+		name, DAOS_VERSION, DAOS_API_VERSION_MAJOR,
+		DAOS_API_VERSION_MINOR, DAOS_API_VERSION_FIX);
+	fprintf(stdout, "Using fuse %s\n", fuse_pkgversion());
+};
+
 static void
 show_help(char *name)
 {
-	printf("usage: %s -m=PATHSTR -s=RANKS\n"
+	printf("usage: %s -m mountpoint\n"
+		"Options:\n"
 		"\n"
-		"	-m --mountpoint=PATHSTR	Mount point to use\n"
-		"	   --pool=<name>	pool UUID/label\n"
-		"	   --container=<name>	container UUID/label\n"
+		"	-m --mountpoint=<path>	Mount point to use\n"
+		"\n"
+		"	   --pool=label		pool UUID/label\n"
+		"	   --container=label	container UUID/label\n"
 		"	   --sys-name=STR	DAOS system name context for servers\n"
-		"	-S --singlethreaded	Single threaded\n"
-		"	-t --thread-count=COUNT Number of fuse threads to use\n"
+		"\n"
+		"	-S --singlethread	Single threaded\n"
+		"	-t --thread-count=count	Number of fuse threads to use\n"
 		"	-f --foreground		Run in foreground\n"
-		"          --disable-caching    Disable all caching\n"
-		"          --disable-wb-cache   Use write-through rather than write-back cache\n",
-		name);
+		"	   --disable-caching	Disable all caching\n"
+		"	   --disable-wb-cache	Use write-through rather than write-back cache\n"
+		"\n"
+		"	-h --help		Show this help\n"
+		"	-v --version		Show version\n"
+		"\n"
+		"Specifying pool and container are optional, if not set then dfuse can connect to\n"
```

Comment:
Grammar nitpick: I don't believe it's "proper" to use a commas here (comma splicing). Generally, this should be two sentences or use a semicolon. Though, I'm not sure if this differs between American and British English.
```suggestion
		"Specifying pool and container are optional; if not set then dfuse can connect to\n"
```
```suggestion
		"Specifying pool and container are optional. If not set then dfuse can connect to\n"
```

Reply:
Done.